test/unit/component.test.js

```diff
@@ -111,6 +112,16 @@ test('should do a deep merge of child options', function(){
   Component.prototype.options_ = null;
 });
 
+test('should init child components from component options', function(){
+  var comp = new ControlBar(TestHelpers.makePlayer(), {
```

Comment:
It's a little weird that component.test now depends on a subclass of component. An alternative (that might not be worth it) would be to define some very simple components in this file instead of re-using ControlBar, VolumeControl, and VolumeMenuButton.